### PR TITLE
[ENG-6672] |  fixed bug when workspace item_type selected for Bitbucket

### PR DESCRIPTION
**Pull request by sh-andriy**
`[ENG-6672]` |  fixed bug when workspace item_type selected for Bitbucket

[ENG-6672]: https://openscience.atlassian.net/browse/ENG-6672?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: addon_imps/storage/bitbucket.py [hunk 1]
```diff
@@ -204,11 +209,13 @@ async def _list_repository_child_items(
                 continue
             item_name = path.split("/")[-1] or "Unnamed Item"
             item_id_value = self._make_item_id("repository", f"{repo_full_name}/{path}")
+            can_be_root_val = item_type_value == storage.ItemType.FOLDER
```
Comment (opaduchak) [inline addon_imps/storage/bitbucket.py:212]: Waterbutler doesn't support setting folder inside repositoey as bibucket root (as with all git-like addons)

File: addon_imps/storage/bitbucket.py [hunk 2]
```diff
@@ -173,6 +177,7 @@ async def _list_workspace_child_items(
                     item_id=item_id,
                     item_name=name,
                     item_type=storage.ItemType.FOLDER,
+                    can_be_root=True,
```
Comment (opaduchak) [inline addon_imps/storage/bitbucket.py:180]: this is default behaviour, you needn't have explicitly stated this

File: addon_imps/storage/bitbucket.py [hunk 3]
```diff
@@ -121,10 +123,12 @@ async def get_item_info(self, item_id: str) -> storage.ItemResult:
                         else storage.ItemType.FILE
                     )
                     item_name = path_param.split("/")[-1] or "Unnamed Item"
+                    can_be_root_val = item_type_value == storage.ItemType.FOLDER
```
Comment (opaduchak) [inline addon_imps/storage/bitbucket.py:126]: Waterbutler doesn't support setting folder inside repository as bitbucket root (as with all git-like addons)

File: addon_imps/storage/bitbucket.py [hunk 4]
```diff
@@ -110,6 +111,7 @@ async def get_item_info(self, item_id: str) -> storage.ItemResult:
                         item_id=item_id,
                         item_name=name,
                         item_type=storage.ItemType.FOLDER,
+                        can_be_root=True,
```
Comment (opaduchak) [inline addon_imps/storage/bitbucket.py:114]: this is default behaviour, you needn't have explicitly stated this

**Review by opaduchak — CHANGES_REQUESTED** [review]
Please adjust boolean flags a litte, rather than that everything is good

**Review by opaduchak — APPROVED** [review]
LGTM

**Review by brianjgeiger — APPROVED** [review]

**Pull request merged**